pom.xml

```diff
@@ -100,16 +100,28 @@
         </exclusion>
       </exclusions>
     </dependency>
-
     <!-- Needed for audit functionality -->
-    <!--
     <dependency>
```

Comment:
Oops, my github did not update with your last 2 messages. But yes, as I understand it you are correct.